fdbserver/DDShardTracker.actor.cpp

```diff
@@ -937,12 +937,12 @@ ACTOR Future<Void> brokenPromiseToReady(Future<Void> f) {
 }
 
 static bool shardMergeFeasible(DataDistributionTracker* self, KeyRange const& keys, KeyRangeRef adjRange) {
```

Comment:
Would it make sense to rename this function to clarify that it is only looking at whether the merge is feasible from a tenant perspective? (For example, something like `tenantAwareShardMergeFeasible` or `shardMergeFeasibleForTenants`.) This helps distinguish it from other reasons why merge may [not] be feasible, like the key-range over-replication that Evan is adding right now.

Reply:
Yeah, I think that sounds good. It's probably out of scope of this PR, but it's a good change to make in a later one.